pkgs/desktops/gnome/misc/gnome-epub-thumbnailer/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv
+, lib
+, fetchurl
+, meson
+, ninja
+, pkg-config
+, gnome
+, gdk-pixbuf
+, glib-networking
+, libarchive
+, libxml2
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-epub-thumbnailer";
+  version = "1.7";
+
+  src = fetchurl {
+    url = "mirror://gnome/sources/gnome-epub-thumbnailer/${lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
+    sha256 = "sha256-S7Ah++RCgNuY3xTBH6XkMgsWe4GpG9e6WGvqDE+il1I=";
+  };
+
+  nativeBuildInputs = [
+    libxml2
+    meson
+    ninja
+    pkg-config
+  ];
+
+  buildInputs = [
```

Comment:
And `librsvg` here for svg covers:

https://gitlab.gnome.org/GNOME/gnome-epub-thumbnailer/-/blob/24829a20630ee5a004f00b5fdd3b6830f593503f/src/gnome-epub-thumbnailer.c#L304

Reply:
SVG generation work correctly without this dependency, unsure where it pulls it in.

Reply:
I meant for epubs with SVG file as a cover.

Not sure what you mean by SVG generation – thumbnailers car really only generate PNGs.